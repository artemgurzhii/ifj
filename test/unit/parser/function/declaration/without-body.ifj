Declare Function foo(n as integer) As Integer 
