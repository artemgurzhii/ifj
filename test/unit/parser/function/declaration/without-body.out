(declare
  (function foo -> (id integer)
  (decl
    (id n)
    : (id integer))
)
